siriuspy/siriuspy/pwrsupply/csdev.py

```diff
@@ -1306,26 +1368,40 @@ def _get_ps_FBP_propty_database():
 def _get_ps_FBP_DCLink_propty_database():
     """Return database with FBP_DCLink pwrsupply model PVs."""
     propty_db = _get_ps_common_propty_database()
+    # FBP DCLinks have different units for Voltage-* PVs
     db_ps = {
-        'Voltage-SP': {'type': 'float', 'value': 0.0,
-                       'lolim': 0.0, 'hilim': 100.0, 'prec': 4},
-        'Voltage-RB': {'type': 'float', 'value': 0.0,
-                       'lolim': 0.0, 'hilim': 100.0, 'prec': 4},
-        'VoltageRef-Mon': {'type': 'float', 'value': 0.0,
-                           'lolim': 0.0, 'hilim': 100.0, 'prec': 4},
-        'Voltage-Mon': {'type': 'float', 'value': 0.0, 'prec': 4},
-        'Voltage1-Mon': {'type': 'float', 'value': 0.0, 'prec': 4},
-        'Voltage2-Mon': {'type': 'float', 'value': 0.0, 'prec': 4},
-        'Voltage3-Mon': {'type': 'float', 'value': 0.0, 'prec': 4},
-        'VoltageDig-Mon': {'type': 'int', 'value': 0,
-                           'lolim': 0, 'hilim': 255},
-        'IntlkSoftLabels-Cte':  {'type': 'string',
-                                 'count': len(_et.SOFT_INTLCK_FBP_DCLINK),
-                                 'value': _et.SOFT_INTLCK_FBP_DCLINK},
-        'IntlkHardLabels-Cte':  {'type': 'string',
-                                 'count': len(_et.HARD_INTLCK_FBP_DCLINK),
-                                 'value': _et.HARD_INTLCK_FBP_DCLINK},
-        'ModulesStatus-Mon': {'type': 'int', 'value': 0},
+        'Voltage-SP': {
+            'type': 'float', 'value': 0.0,
+            'lolim': 0.0, 'hilim': 100.0, 'prec': 4, 'unit': 'A'},
+        'Voltage-RB': {
+            'type': 'float', 'value': 0.0,
+            'lolim': 0.0, 'hilim': 100.0, 'prec': 4, 'unit': 'A'},
+        'VoltageRef-Mon': {
+            'type': 'float', 'value': 0.0,
+            'lolim': 0.0, 'hilim': 100.0, 'prec': 4, 'unit': 'A'},
+        'Voltage-Mon': {
+            'type': 'float', 'value': 0.0, 'prec': 4, 'unit': 'A'},
+        'Voltage1-Mon': {
+            'type': 'float', 'value': 0.0, 'prec': 4, 'unit': 'A'},
+        'Voltage2-Mon': {
+            'type': 'float', 'value': 0.0, 'prec': 4, 'unit': 'A'},
+        'Voltage3-Mon': {
+            'type': 'float', 'value': 0.0, 'prec': 4, 'unit': 'A'},
+        'VoltageDig-Mon': {
+            'type': 'int', 'value': 0,
+            'lolim': 0, 'hilim': 255, 'unit': 'A'},
```

Comment:
I think the correct unit here is `V`

Reply:
I am not sure, Ana. I think for FBP DCLinks units are different from other models for these PVs. I have commented it , since they operate with different `CtrlLoop` value. But I am not sure whether this is the case only for `Voltage-SP|RB` or for other voltages, as well. I'll check...